components/QrCodeOverlay.tsx

```diff
@@ -37,31 +37,23 @@ export const QrCodeOverlay: React.FC<QrCodeOverlayProps> = props => {
     return qrData;
   }
 
-  const [base64String, setBase64String] = useState('');
-
   let qrRef = useRef(null);
 
-  function updateBase64() {
+  function sharedQRCodeTapped() {
```

Comment:
There is a button which says `Share QR Code`
so let's keen function name as `onShareQRCodeClick`  or `onShareQRCodePress`

Reply:
Done

Reply:
changed to handle